Join text menus on the top-left, move shortcut buttons to new locations (Preliminary work towards achieving the changes discussed in #8865.)

nunodonato opened the pull request:
Let's test the proposed changes in #8865 so far and see if the new locations for the menus and buttons are a good idea.

Note: The old stuff in the top-right (volume, updates, etc), are still being created but simply not added to the GUI. This is because it needs to be disconnected first from other places inside Godot, otherwise it will crash.

nunodonato committed:
Join text menus on the top-left, move shortcut buttons to new location

HummusSamurai commented:
Relevant [comment](https://github.com/godotengine/godot/issues/8865#issuecomment-303661634) from the referenced mega-thread.

nunodonato commented:
Just updated this PR with more changes

- "remote" button now became "debug" text menu
- removed the volume indicator on the right, but kept the update spinner, for now

eon-s commented:
Even if for some people those options are not "debug" proper (and maybe are not), I think devs coming from other engines (box2d, haxe engines, Unity and more) will understand it better with that name.

akien-mga commented:
Let's merge that current state, and see if/how we want to improve it further.

RodeoMcCabe commented:
This looks great. The top has really been cleaned up, which IMO needed doing. The update spinner hardly takes any room and is kind of neat, though it might not fit in with the rest of the icons on the right.

Thanks for your work!
